### ISSUE TITLE
Uncaught InvalidStateError: Failed to execute 'setSelectionRange' on 'HTMLInputElement': The input element's type ('number') does not support selection.

### ISSUE THREAD

**Issue by mrdulin** [Lucifier129/react-lite#131]
![image](https://user-images.githubusercontent.com/17866683/35139933-3b8d4d14-fd30-11e7-8eaa-513ab2629c22.png)

`Uncaught InvalidStateError: Failed to execute 'setSelectionRange' on 'HTMLInputElement': The input element's type ('number') does not support selection.`

`"react-lite": "^0.15.38",`
`"fastclick": "^1.0.6",`

google chrome:  63.0.3239.132, device mode

related issue: https://github.com/ftlabs/fastclick/issues/358

**Comment by Lucifier129** [comment]
It seems like this is not a problem with `react-lite`, I have no idea what can we do.

**Comment by mrdulin** [comment]
@Lucifier129 Yeah, but `react-lite/lib/react-tap-event-plugin.js` use `fastclick`, it's a `fastclick` issue.
Use `react-tap-event-plugin`, the error gone.
